### PR TITLE
Update executor-types.md

### PR DESCRIPTION
**Pull request by iynere**
```
machine: true
  image: etc
```

isn't valid yaml! it's gotta be:

```
machine:
  enabled: true
  image: etc
```

### CODE REVIEW

File: jekyll/_cci2/executor-types.md [hunk 1]
```diff
@@ -59,7 +59,8 @@ The following example specifies the default `machine` image.
 ```YAML
 jobs:
   build:
-    machine: true
+    machine:
+      enabled: true
```
Comment (ndintenfass) [inline jekyll/_cci2/executor-types.md]: Pretty sure you don't need the "enabled" key here.

Comment (iynere) [reply jekyll/_cci2/executor-types.md]: oh tru !! that makes sense, so it's more like the docker syntax

**Comment by iynere** [comment]
incorporated nathan's feedback & cleaned up wording

**Comment by eric-hu** [comment]
LGTM

**Pull request merged**